common/CMakeLists.txt

```diff
@@ -15,15 +15,21 @@ include(GNUInstallDirs)
 # Default values
 set(BEEROCKS_BRIDGE_IFACE "br-lan")
 set(BEEROCKS_PLATFORM_INIT "false")
-set(BEEROCKS_INSTALL_PATH "/opt/prplmesh")
+set(BEEROCKS_INSTALL_PATH "/opt/prplMesh")
 set(BEEROCKS_TMP_PATH "/tmp/beerocks")
 set(BEEROCKS_WLAN1_IFACE "wlan0")
 set(BEEROCKS_WLAN2_IFACE "wlan2")
+set(BEEROCKS_BH_WIRE_IFACE "eth1")
 
 # Platform specific overrides
-if (TARGET_PLATFORM STREQUAL "ugw")
-    add_definitions(-DBEEROCKS_UGW)
-    set(BEEROCKS_BH_WIRE_IFACE "eth1")
+if (TARGET_PLATFORM STREQUAL "openwrt")
+    # UGW
+    if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
+        add_definitions(-DBEEROCKS_UGW)
```

Comment:
It's about time we get rid of BEEROCKS_UGW. it is only used on beerocks_version - so OK, for now, to keep it only for version print.

Other uses - 

- network_utils.cpp - used for endianess in `mac_to_uint64` and broken for none UGW anyway (need to remove BEEROCKS_BRCM support here)
- BWL DWPAL - used for different safec libraries includes. It would be better to have a different h file which will include the correct header based on the safec library found by the BWL's CMakeLists.txt
- BPL UCI - same as BWL comment